src/doc-utils.js

```diff
@@ -33,6 +33,23 @@ function traverseDoc(doc, onEnter, onExit) {
   traverseDocRec(doc);
 }
 
+function mapDoc(doc, func) {
+  doc = func(doc);
+
+  if(doc.type === "concat") {
```

Comment:
Thanks for clarifying. I don't personally share this view (even in other languages) but everyone has their preferences! :+1: